src/main/java/org/broadinstitute/hellbender/tools/funcotator/FilterFuncotations.java

```diff
@@ -143,12 +143,12 @@ public void apply(final VariantContext variant, final ReadsContext readsContext,
                 funcotationKeys, variant, "Gencode_" + reference.gencodeVersion + "_annotationTranscript", "FAKE_SOURCE");
 
         funcs.values().forEach(funcotationMap -> {
-            final Stream<Map<String, String>> transcriptFuncotations = funcotationMap.getTranscriptList().stream()
+            final Stream<Set<Map.Entry<String, String>>> transcriptFuncotations = funcotationMap.getTranscriptList().stream()
```

Comment:
With multi-allelic variants, we were getting errors because the same key was being seen twice. Using a `Set<Map.Entry>` means we can collect all the values based on the hash of the key-value pair, even if the keys are the same.

Reply:
Ah!  Cool.